MANUAL_REGRESSION.md

```diff
@@ -113,6 +113,13 @@ Go through the flow looking for layout/usability inconsitencies between browsers
 3. Upload face photo
     - everything should be displayed properly and layout should not be broken
 
+##### 9. Check the camera is mirroring
+(on an iOS and Android device; a laptop with camera)
```

Comment:
I was not able to reproduce non-mirrored camera feed on laptop with camera. I was able to reproduce only on third-party camera, hence I would suggest to change the instruction to `; a third-party USB camera`

Reply:
I'm able to see it not mirrored on latest on my macbook pro :/ and since this test is to confirm it mirrors, I think it makes sense to add  `third party usb camera`, but not to replace.

Reply:
Yup in that case fair enough